### ISSUE TITLE
Reproduce results of SMI

### ISSUE THREAD

**Issue by xingpingdong** [Hugo101/PLATINUM#4]
Hi,
I tried to reproduce the results reported in your paper. Thus, I tried to use the commands as follows.

python maml_ssl_main.py --dataset miniimagenet --ratio 0.0333 --scenario woDistractor --ssl_algo SMI --budget_q 50

But I only get the test accuracy of 0.2687 on 5 way 1 shot task after 300 epochs and I found that the validation accuracy did not improve after 100 epochs. I think this may be caused by the wrong hyperparameters. Could you share your hyperparameters to reproduce the results on the paper?

By the way, I change the code to use the validation dataset to select the best model.

Thank you very much!
